aiven/datasource_elasticsearch_acl.go

```diff
@@ -11,9 +11,9 @@ import (
 
 func datasourceElasticsearchACL() *schema.Resource {
 	return &schema.Resource{
-		ReadContext: datasourceElasticsearchACLRead,
-		Schema: resourceSchemaAsDatasourceSchema(aivenElasticsearchACLSchema,
-			"project", "service_name"),
+		DeprecationMessage: "ElasticsearchACL is deprectated as it is not easily modularizable, please use ElasticsearchACLConfig and ElasticsearchACLSet",
```

Comment:
Perhaps we don't have to provide such a long deprecation message, for my money `ElasticsearchACL is deprecated please use ElasticsearchACLConfig and ElasticsearchACLSet` should do. And btw there is a small typo in word `deprectated`

Reply:
Thanks! I will update the message and fix the typo

Reply:
done